src/components/__tests__/BadgeAchievement.cy.js

```diff
@@ -1,7 +1,11 @@
+import { colors } from 'quasar';
+
 import BadgeAchievement from '../BadgeAchievement.vue';
 import { i18n } from '../../boot/i18n';
 import { badgeList } from '../../mocks/homepage';
 
+const { getPaletteColor } = colors;
+
```

Comment:
Better way how to declare same color for one component test 

```ts
const grey9 = getPaletteColor('grey-9');
```

Reply:
Added. I extended this to all the uses to indicate correct use across repo.